From fef162b775ac00897f65f79855c1d9cf0bf5c7e2 Mon Sep 17 00:00:00 2001
From: Andrew Boie <andrew.p.boie@intel.com>
Date: Thu, 8 Aug 2013 14:36:03 -0700
Subject: REVERTME: increase search depth to 6

In upstream code review Google insisted on a max search depth of
4, but currently some product definitions live deeper than that.
Increase to 6 until we can sort this out.

JIRA to address this issue:
https://otci.intel.com/jira/browse/ADEV-166

Bug: ADEV-107
Domain: AOSP-build
Category: Device Enablement
Source: internal
Upstream-candidate: No

Change-Id: If0eb868c0ca2d78bbe78297895f8042b47d55720
Signed-off-by: Andrew Boie <andrew.p.boie@intel.com>
---
 core/config.mk |    4 ++--
 envsetup.sh    |    4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/core/config.mk b/core/config.mk
index d49e794..f9afadb 100644
--- a/core/config.mk
+++ b/core/config.mk
@@ -143,8 +143,8 @@ include $(BUILD_SYSTEM)/envsetup.mk
 board_config_mk := \
 	$(strip $(wildcard \
 		$(SRC_TARGET_DIR)/board/$(TARGET_DEVICE)/BoardConfig.mk \
-		$(shell test -d device && find device -maxdepth 4 -path '*/$(TARGET_DEVICE)/BoardConfig.mk') \
-		$(shell test -d vendor && find vendor -maxdepth 4 -path '*/$(TARGET_DEVICE)/BoardConfig.mk') \
+		$(shell test -d device && find device -maxdepth 6 -path '*/$(TARGET_DEVICE)/BoardConfig.mk') \
+		$(shell test -d vendor && find vendor -maxdepth 6 -path '*/$(TARGET_DEVICE)/BoardConfig.mk') \
 	))
 ifeq ($(board_config_mk),)
   $(error No config file found for TARGET_DEVICE $(TARGET_DEVICE))
diff --git a/envsetup.sh b/envsetup.sh
index 0661fe6..cddd15f 100644
--- a/envsetup.sh
+++ b/envsetup.sh
@@ -1333,8 +1333,8 @@ if [ "x$SHELL" != "x/bin/bash" ]; then
 fi
 
 # Execute the contents of any vendorsetup.sh files we can find.
-for f in `test -d device && find device -maxdepth 4 -name 'vendorsetup.sh' 2> /dev/null` \
-         `test -d vendor && find vendor -maxdepth 4 -name 'vendorsetup.sh' 2> /dev/null`
+for f in `test -d device && find device -maxdepth 6 -name 'vendorsetup.sh' 2> /dev/null` \
+         `test -d vendor && find vendor -maxdepth 6 -name 'vendorsetup.sh' 2> /dev/null`
 do
     echo "including $f"
     . $f
-- 
1.7.4.1

